src/pages/index.tsx

```diff
@@ -2,19 +2,15 @@ import { ReactElement } from 'react'
 import BaseNewLayout from 'layouts/base-new'
 import ProductCardGrid from 'components/product-card-grid'
 import HomepageHero from 'components/homepage-hero'
-import Footer from 'components/footer'
```

Comment:
`Footer` now handled by `BaseNewLayout`. I think I might've handled this in the Vault `/docs` PR as well, but hoping to merge this PR first, I think, so will rebase the `/docs` work onto this.